### PR TITLE
chore: avatar loading tweaks

### PR DESCRIPTION
**Pull request by AjimenezDCL**
#332 

The spinner in the avatar editor is gone and the transition when loading wearables in smoother

### CODE REVIEW

**Comment by AjimenezDCL** [comment]
https://play.decentraland.zone/index.html?ENV=org&position=0%2C-1&realm=artemis-amber&renderer=urn%3Adecentraland%3Aoff-chain%3Arenderer-artifacts%3Achore%2Favatar_loading_tweaks

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/ParticlesOnAvatarVisualCue.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    [SerializeField] private AvatarRenderer.VisualCue avatarVisualCue;
+    [SerializeField] private GameObject particlePrefab;
+    [SerializeField] private bool followAvatar;
+
+    private AvatarRenderer avatarRenderer;
+    private void Awake()
+    {
+        avatarRenderer = GetComponent<AvatarRenderer>();
+        if (avatarRenderer == null)
+            return;
```
Comment (sandrade-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/ParticlesOnAvatarVisualCue.cs:16]: As the class already have the `[RequireComponent(typeof(AvatarRenderer))]`, it is not necessary to check if the `AvatarRenderer` component exists, isn't it?

**Review by sandrade-dcl — APPROVED** [review]
LGTM! I only left a little comment 👍

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/ParticlesOnAvatarVisualCue.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/ParticlesOnAvatarVisualCue.cs:16]: You are completly right, thanks!

**Review by pbosio — COMMENTED** [review]
All visual indicators that the wearable is loading on the avatar were removed?
Right now if you click "Randomize" it feels that the button do nothing until several seconds later when you can see that wearables on the avatar has changed

**Comment by AjimenezDCL** [comment]
@pbosio 
> All visual indicators that the wearable is loading on the avatar were removed?
> Right now if you click "Randomize" it feels that the button does nothing until several seconds later when you can see that wearables on the avatar have changed

That's a good point, I'm coming back to @RominaMarchetti and @ShibuyaMartin to solve it before merging this.

**Comment by AjimenezDCL** [comment]
@pbosio @pravusjif we are gonna address your concerns in this PR, I will request your review back when it's done. Thanks

**Review by pbosio — APPROVED** [review]

**Pull request merged**